tests/phpunit/CRM/Core/TransactionTest.php

```diff
@@ -362,6 +362,7 @@ public function createContactWithTransaction($nesting, $insert, $outcome) {
 
     if ($insert == 'sql-insert') {
       $r = CRM_Core_DAO::executeQuery("INSERT INTO civicrm_contact(first_name,last_name) VALUES ('ff', 'll')");
+      // @fixme mysql_insert_id() does not exist in PHP7.
```

Comment:
it exists, see https://github.com/mollux/civicrm-core/commit/7e5e2fd51e0b08af24bbedbb1a06d127410f560e#diff-99201c02152b6d59169f6cd6c90b04c0L365